packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -18,19 +19,24 @@ import {
     ITestContainerConfig,
     DataObjectFactoryType,
 } from "@fluidframework/test-utils";
-import {
-    describeFullCompat,
-} from "@fluidframework/test-version-utils";
+import { describeFullCompat } from "@fluidframework/test-version-utils";
+import { ConfigTypes, IConfigProviderBase } from "@fluidframework/telemetry-utils";
 
 const map1Id = "map1Key";
 const map2Id = "map2Key";
 const registry: ChannelFactoryRegistry = [
     [map1Id, SharedMap.getFactory()],
     [map2Id, SharedMap.getFactory()],
 ];
+const configProvider = ((settings: Record<string, ConfigTypes>): IConfigProviderBase => {
+    return {
+        getRawConfig: (name: string): ConfigTypes => settings[name],
+    };
+});
 const testContainerConfig: ITestContainerConfig = {
     fluidDataObjectType: DataObjectFactoryType.Test,
     registry,
+    loaderProps: { configProvider: configProvider({ "Fluid.ContainerRuntime.FlushModeTurnBased": "true" }) },
```

Comment:
This feature gate doesn't exist anymore, so the container is `FlushMode.TurnBased` by default.

Reply:
Ah. I was working off of an older repo!